www/src/data/sidebars/doc-links.yaml

```diff
@@ -366,6 +366,9 @@
             - title: Adding Forms
               link: /docs/adding-forms/
               breadcrumbTitle: Forms
+            - title: Adding React Components
```

Comment:
Let's hear what other members of @gatsbyjs/learning have to weight in on where this doc would go. Seems like this is a valuable and important concept and may even need to be higher up in the sidebar somewhere.

Reply:
I think this is a fine place to start.